primitives/src/v2/mod.rs

```diff
@@ -1403,6 +1403,25 @@ pub struct DisputeState<N = BlockNumber> {
 	pub concluded_at: Option<N>,
 }
 
+#[cfg(feature = "std")]
+impl MallocSizeOf for DisputeState {
+	fn size_of(&self, ops: &mut MallocSizeOfOps) -> usize {
+		// destructuring to make sure no new fields are added to the struct without modifying this function
+		let Self { validators_for, validators_against, start, concluded_at } = self;
+
+		// According to the documentation `.capacity()` might not return a byte aligned value, so just in case:
+		let align_eight = |d: usize| match d % 8 {
+			0 => d / 8,
+			_ => d / 8 + 1,
+		};
```

Comment:
```suggestion
		let align_eight = |d: usize| (d + 7) % 8;
```

Reply:
Shouldn't it be `let align_eight = |d: usize| (d + 7) / 8;` ?

Reply:
yes, indeed, you're correct!

Reply:
We can just use / 8 for simplicity. It's not precise either way, nor is it supposed to be.

Reply:
Yeah, but it's a simple fix and pretty much for free, and we should prefer the upper bound rather than not.